src/main/kotlin/io/github/moulberry/notenoughupdates/miscfeatures/profileviewer/SacksPage.kt

```diff
@@ -321,6 +419,18 @@ class SacksPage(pvInstance: GuiProfileViewer) : GuiProfileViewerPage(pvInstance)
             }
         }
 
+        if (priceSourceButtonRect.contains(mouseX, mouseY)) {
+            currentPriceSource = PriceSource.values()[(currentPriceSource.ordinal + 1) % PriceSource.values().size]
+            Utils.playPressSound()
+            getData()
+        }
+
+        if (sortButtonRect.contains(mouseX, mouseY)) {
```

Comment:
At this point we might as well add a utility like
```kt

    inline fun <reified E : Enum<E>> E.nextEnum(): E {
        val values = enumValues<E>()
        return values[(ordinal + 1) % values.size]
    }

```

Not really needed here tho.